clients:
  prolog_query:
    service: "json_prolog/simple_query"
  prolog_next_solution:
    service: "json_prolog/next_solution"
  prolog_finish:
    service: "json_prolog/finish"

publishers:
  interactive_marker:
    topic: "~markers"
    queue_size: 1
  semantic_map_action_feedback:
    topic: "~actions"
    queue_size: 1
    latch: false

interactive_markers:
  handles:
    enabled: true
    scale: 0.3
    embedded_materials: false
    color:
      r: 1.0
      g: 0.0
      b: 0.0
      a: 1.0
